core/validation/pre_transformation_validate.py

```diff
@@ -0,0 +1,148 @@
+    """
+    if reporting_round in [1, 2]:
+        return  # do not do pre-transformation validation on historical data
+
+    if reporting_round == 4:
+        authorisation_validation(workbook, place_names, fund_types)  # return first as unauthorised cannot be validated
+    wrong_input_validation(workbook, reporting_round)
+    conflicting_input_validation(workbook)  # return last as invalid input should not be
+    # returned with conflicting input
+
+
+def authorisation_validation(workbook: dict[str, pd.DataFrame], place_names: tuple, fund_types: tuple):
+    """
+    Validate authorisations for the given workbook.
+
+    This function checks if the submitted workbook has the required authorisation for place names and fund types.
+
+    :param workbook: A dictionary containing sheets of the workbook.
+    :param place_names: A tuple of place names for validation.
+    :param fund_types: A tuple of fund types for validation.
+    :raises ValidationError: If authorisation validation fails, raises a ValidationError with associated failures.
+    """
+    authorisation_checks = {}
+    sheet = workbook.get("2 - Project Admin")
+
+    if str(sheet.iloc[6][4]).strip() in TF_PLACE_NAMES_TO_ORGANISATIONS.keys():
+        authorisation_checks["Place Name"] = (
+            str(sheet.iloc[6][4]).strip(),
+            place_names,
+        )
+    if FUND_TYPE_TO_TD_OR_HS.get(sheet.iloc[5][4], sheet.iloc[5][4]) in {"TD", "HS"}:
+        authorisation_checks["Fund Type"] = (
+            FUND_TYPE_TO_TD_OR_HS.get(sheet.iloc[5][4], sheet.iloc[5][4]),
+            fund_types,
+        )
+
+    failures = check_values(authorisation_checks, "unauthorised_submission_failure")
+
+    if failures:
+        raise ValidationError(validation_failures=failures)
+
+
+def wrong_input_validation(workbook: dict[str, pd.DataFrame], reporting_round: int):
+    """
+    Validate wrong input in the given workbook for the specified reporting round.
+
+    :param workbook: A dictionary containing sheets of the workbook.
+    :param reporting_round: An integer representing the reporting round for the validation.
+    :raises ValidationError: If wrong input validation fails, raises a ValidationError with associated failures.
+    """
+    form_version, reporting_period = GET_FORM_VERSION_AND_REPORTING_PERIOD[reporting_round]
+
+    wrong_input_checks = {}
+    sheet1 = workbook.get("1 - Start Here")
+    sheet2 = workbook.get("2 - Project Admin")
+
+    wrong_input_checks["Form Version"] = (
+        sheet1.iloc[6][1],
+        (form_version,),
+    )
+    wrong_input_checks["Reporting Period"] = (sheet1.iloc[4][1], (reporting_period,))
+    wrong_input_checks["Fund Type"] = (
+        FUND_TYPE_TO_TD_OR_HS.get(sheet2.iloc[5][4], sheet2.iloc[5][4]),
+        ("TD", "HS"),
+    )
+    wrong_input_checks["Place Name"] = (
+        str(sheet2.iloc[6][4]).strip(),
+        tuple(TF_PLACE_NAMES_TO_ORGANISATIONS.keys()),
+    )
```

Comment:
This is still very specific to the TF spreadsheet.. can we pull this out as config.. something like:

```
Check = namedtuple("Check", ("sheet", "column", "row", "expected_values", "type"))

TF_ROUND_4 = {
    "checks": (
        Check(sheet="1 - Start Here", column=6, row=1, expected_values=("v4.3",)),
        Check(sheet="1 - Start Here", column=4, row=1, expected_values=("1 April 2023 to 30 September 2023",)),
        Check(sheet="2 - Project Admin", column=5, row=4, expected_values=("TD", "HS")),
        Check(
            sheet="2 - Project Admin", column=6, row=4, expected_values=tuple(TF_PLACE_NAMES_TO_ORGANISATIONS.keys())
        ),
    )
}
```